docs/resources/metric_tag_configuration.md

```diff
@@ -49,6 +50,7 @@ resource "datadog_metric_tag_configuration" "example_count_metric" {
 ### Optional
 
 - `aggregations` (Block Set) A list of queryable aggregation combinations for a count, rate, or gauge metric. By default, count and rate metrics require the (time: sum, space: sum) aggregation and gauge metrics require the (time: avg, space: avg) aggregation. Can only be applied to metrics that have a `metric_type` of count, rate, or gauge. (see [below for nested schema](#nestedblock--aggregations))
+- `exclude_tags_mode` (Boolean) Toggle to include/exclude tags as queryable for your metric.  Defaults to false.  Can only be applied to metrics that have one or more tags configured. Defaults to `false`.
```

Comment:
Should this match the same text as what we have above for the `exclude_tags_mode` description? also can we remove the double spaces around `Defaults to false.`